ota-plus-web/app/reactapp/src/stores/UserStore.js

```diff
@@ -53,11 +55,28 @@ export default class UserStore {
     resetAsync(this.userChangePasswordAsync);
     resetAsync(this.contractsFetchAsync);
     resetAsync(this.contractsAcceptAsync);
+    resetAsync(this.editOrganizationNameAsync);
     resetAsync(this.getOrganizationsAsync);
     resetAsync(this.getOrganizationUsersAsync);
     resetAsync(this.addUserToOrganizationAsync);
   }
 
+  editOrganizationName = async (namespace, name) => {
+    const namespaceCookie = Cookies.get(ORGANIZATION_NAMESPACE_COOKIE);
+    // just to be sure user edits his won organization
+    if (namespace === namespaceCookie) {
```

Comment:
What do we need this check at all? The `PATCH` endpoint will use the current endpoint. I would say that this `editOrganizationName` method should only take the new name as parameter.

Reply:
Yes, indeed. It was before the endpoint has changed. Done.